Resources/public/templates/tabs/locations.hbt

```diff
@@ -25,8 +25,17 @@
                                     <a href="{{ path "viewLocation" id=id languageCode=contentInfo.mainLanguageCode }}">{{ pathString }}</a>
                                 </td>
                                 <td class="ez-table-data-childcount">{{ childCount }}</td>
-                                <td class="ez-table-data-visibility">
-                                    {{#if hidden}}Hidden{{else}}{{#if invisible}}Hidden by superior{{else}}Visible{{/if}}{{/if}}
+                                <td class="ez-table-data-visibility pure-g">
```

Comment:
you don't need a grid here, the visibility followed by the button would do the trick.